ui/src/Live/Multiview.tsx

```diff
@@ -198,15 +200,18 @@ interface MonoviewProps {
 
 /** A single pane of a Multiview, including its camera chooser. */
 const Monoview = (props: MonoviewProps) => {
+  const [searchParams, setSearchParams] = useSearchParams();
```

Comment:
A couple things I noticed trying this out:

*   It looks like the behavior on initial page load is different now: before you'd see `(none)` in the selector; now you see nothing at all (but still (none) in the pull-down which doesn't match it).
*   It doesn't work properly when displaying more than one camera at a time. (That is: after going up to the menu bar and changing "solo" to something else.) Probably the search params should be on `Multiview` rather than `Monoview` and parse a comma-separated list.

Reply:
I move that to MultiView, and handle selection layout. But, i sets params on some "change" event. When some one, set layout, and back to "list" a one again come to "live", current (old - because react keep it in store) wasn't updated to url - until user change once again camera.

I think, should be better if we can store saved layout in localStorage, and genere links by clicking button like "copy url". That would be better